src/components/Dashboard/CustomTabs/CustomTabs.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+// nodejs library that concatenates classes
+import classNames from 'classnames';
+// nodejs library to set properties for components
+
+// material-ui components
+import { makeStyles } from '@material-ui/core/styles';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+// core components
+import { Card } from '../../common';
+import CardBody from '../Card/CardBody';
+import CardHeader from '../Card/CardHeader';
+
+import styles from '../../../assets/jss/material-dashboard-react/components/customTabsStyle';
+import { colors } from '../../../assets/jss/Colors';
+import { SvgIcon } from '@material-ui/core';
+
+const useStyles = makeStyles(styles);
+type CustomTabsProps = {
+  headerColor: colors;
+  title: string;
+  tabs: {
+    tabName: string;
+    tabIcon: any;
+    tabContent: React.ReactNode;
+  }[];
+  rtlActive?: boolean;
```

Comment:
presumably the ? character is part of the variable name here.  I've seen this used to define optional members in interfaces but I guess this is just a concrete type - It makes sense - use a trailing ? for booleans